package.json

```diff
@@ -135,7 +135,6 @@
     "core-js": "^3.18.3",
     "cz-conventional-changelog": "^3.3.0",
     "danger": "^10.6.6",
-    "dateformat": "^5.0.3",
```

Comment:
Are you sure dateformat is no longer needed? Wouldn't it be safer to revert the config.js changes and downgrade dateformat to v4.6.3?

Reply:
I locally reverted to this version. After `npm run build:changelog` I got
```
MacBook:titanium_mobile sebastianklaus$ npm run build:changelog

> titanium-mobile@11.0.0 build:changelog
> conventional-changelog -n changelog/config.js -i CHANGELOG.md -s -p angular

fatal: ambiguous argument 'undefined...HEAD': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'
Failed to get file. Error: Command failed: git log --cherry-pick --right-only --no-merges undefined...HEAD --format=%H
fatal: ambiguous argument 'undefined...HEAD': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'
```

Reply:
Same as for @caspahouzer!

Reply:
I tried with argument branch and tag and release, but maybe I'm just doing it wrong 🥲

Reply:
> fatal: ambiguous argument 'undefined...HEAD': unknown revision

You must have a local branch with previous version. In this case `10_1_X`

https://github.com/tidev/titanium_mobile/blob/f80e1dc35b7e1d8795fd096db2b350fddc94e88c/changelog/config.js#L86

> Are you sure dateformat is no longer needed?

It's not used anywhere, but this place. I think, fewer dependencies, fewer problems.

Reply:
@drauggres is right: if you have a 10_1_X branch it will generate the changelog. 

So:
```
git checkout 10_1_1_GA
git switch -c 10_1_X
npm run build:changelog
```
looks fine. Still needs some manual adjustments (e.g. one of my PRs is listed under `m1ga` and of course a nice intro text or some breaking changes [shouldn't be any in this version]).